redaxo/src/core/lib/login/user_session.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @package redaxo\core\login
+ */
+class rex_user_session
```

Comment:
Gibt es schon Use Cases für Zugriff auf die Klasse aus anderen Addons oder so?
Ansonsten würde ich sie `@internal` setzen, damit wir sie beliebig ändern können.

Reply:
es gibt z.B. Ideen mit dem https://github.com/FriendsOfREDAXO/collaborate, aber ich würde es erstmal als `@internal` markieren.
fix in https://github.com/redaxo/redaxo/pull/5376